README.md

```diff
@@ -1,27 +1,25 @@
-# bpmn-visualization
+<h1 align="center">BPMN Visualization</h1> <br>
+<p align="center">
+   <img title="BPMN Visualization" src="https://github.com/process-analytics/bpmn-visualization-js/blob/master/config/logo.png" width="812" height="170">
+</p>
 
-
-[![Build](https://github.com/process-analytics/bpmn-visualization-js/workflows/Build/badge.svg)](https://github.com/process-analytics/bpmn-visualization-js/actions)
 [![GitHub release (latest by date including pre-releases)](https://img.shields.io/github/v/release/process-analytics/bpmn-visualization-js?color=orange&include_prereleases)](https://github.com/process-analytics/bpmn-visualization-js/releases)
+[![Build](https://github.com/process-analytics/bpmn-visualization-js/workflows/Build/badge.svg)](https://github.com/process-analytics/bpmn-visualization-js/actions)
```

Comment:
:question:  we may probably put the build badge on its own line as it relates to the current build status
Demo and release relate to existing released materials

Reply:
I'm afraid there will too many lines of badges after.
I can put the build badge after the demo badge.